flowcore/src/main/java/com/raizlabs/android/dbflow/converter/BooleanConverter.java

```diff
@@ -11,6 +11,6 @@ public Integer getDBValue(Boolean model) {
 
     @Override
     public Boolean getModelValue(Integer data) {
-        return data != null && data == 1;
+        return data == null ? null : data == 1;
```

Comment:
Yes we may want to return null, however this may break down conversions to ```boolean```. Can you add a test for a regular ```boolean```? thank you